.gitignore

```diff
@@ -17,3 +15,15 @@ fuzz/pulsar-corpus-gen
 fuzz/*.zip
 fuzz/*-workdir/
 fuzz/initial-corpus/
+
+# IDE generated files
+.idea/
+
+# dep generated files
+vendor
```

Comment:
ignore vendor?
what if the original repo was removed or renamed~

Reply:
Later, may consider replacing the `dep` with `go mod`